my-webxr-app/src/data/DataAbstractor.tsx

```diff
@@ -0,0 +1,12 @@
+/**
+ * The DataAbstractor interface defines a set of common operations available to the Data
+ * Abstraction Layer.
+ */
+export default interface DataAbstractor {
+  getAvailableFields: () => Promise<string[]>;
+  storeCSV: (batchItems: Array<Array<string> | Array<number>>) => Promise<boolean>;
```

Comment:
```suggestion
  storeCSV: (batchItems: BatchedDataStream) => Promise<boolean>;
```

Reply:
yes, I will fix this. Thanks so much!!!

Reply:
Ahh, we run into dependency cycle with this.  DataLayer depends on  DataAbstracctor and DataAbstractor depends on BatchedDataStream from Datalayer. Will eslint disable it now

Reply:
In this case, we may be able to break out this type into a new file in the `types` folder. Something we can evaluate in the future.